app/templates/dataset.html

```diff
@@ -53,18 +53,18 @@ <h2>CONP Portal | {{ data.title }} </h2>
   </div>
 </div>
 
+<h2 class="card-title">Download Options</h2>
+Download is enabled through <a style="color: blue;" href="http://handbook.datalad.org/en/latest/intro/installation.html#install">datalad</a>
+<br>
+Here are commands to download content from this dataset:
 <div class="card">
   <div class="card-body">
-    <h2 class="card-title">Download Options</h2>
     <pre class="card-text" style="white-space: pre-wrap;">
-# Download is enabled through <a style="color: blue;" href="http://handbook.datalad.org/en/latest/intro/installation.html#install">datalad</a>
-      
-# Here are commands to download content from this dataset:
-    ~$ datalad install https://github.com/CONP-PCNO/conp-dataset.git
-    ~$ cd conp-dataset
-    ~$ datalad install {{data.id}}
-    ~$ cd {{data.id}}
-    ~$ datalad get &ltfilepath&gt
+datalad install https://github.com/CONP-PCNO/conp-dataset.git
+cd conp-dataset
+datalad install projects/{{data.name}}
```

Comment:
This looks good! The only thing maybe is that the name is only the basename of the directory. However, for projects like openpain subdatasets it should be `projects/openpain/basename` because that is where those are stored. 

Is there a way to include parent directory when there is one in the directive?

Reply:
#225

Reply:
Nice catch!